### ISSUE TITLE
Model is overfitting for Emotion prediction

### ISSUE THREAD

**Issue by shakhyar** [abr-98/Face_data_based_deeplearning#1]
In the emotion detection model, it seems as if your data is overfitting, and your validation loss and accuracy is almost same as training loss and accuracy, so that means you are using the same data for training and validation. Try using more dropouts, and also try using data augmentation techniques to make your model more accurate with unseen images.
